src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,45 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $block-gap;
+  padding: 10px;
+  width: $calendar-width;
+
+  &__block {
+    @include square;
+    display: flex;
+    background-color: #eee;
+    border: 1px solid black;
+    justify-content: center;
+    align-items: center;
+    transition: background-color 0.5s, transform 0.5s ;
+
+    &:hover {
+      background-color: pink;
+      transform: translateY(-20px);
+      cursor: pointer;
+    }
+  }
+
+  @for $index from 1 through 31 {
+    &__block--day--#{$index}::before {
```

Comment:
nth child can help you to avoid this modifier 
![image](https://github.com/mate-academy/layout_calendar/assets/70009205/2d0b42b0-bce8-4e76-8706-cc150c046cea)